GooglePlayInstant/Editor/QuickDeploy/AssetBundleVerifierWindow.cs

```diff
@@ -25,32 +29,44 @@ public class AssetBundleVerifierWindow : EditorWindow
     {
         private const int FieldMinWidth = 170;
 
-        private bool _assetBundleDownloadIsSuccessful;
-        private string _assetBundleUrl;
+        internal bool AssetBundleDownloadIsSuccessful;
+        internal string AssetBundleUrl;
+
         private long _responseCode;
-        private string _errorDescription;
+
+        internal string ErrorDescription;
+
         private string _mainScene;
         private double _numOfMegabytes;
         private UnityWebRequest _webRequest;
+        private AssetBundle _bundle;
+
+        public AssetBundleVerifyState State;
+
+        internal static string WebRequestErrorFormatMessage = "Problem retrieving AssetBundle from {0}: {1}";
```

Comment:
Make const and move up to top with other const fields

Reply:
Resolved.